.github/CODEOWNERS

```diff
@@ -163,8 +163,16 @@
 /tensorflow_addons/optimizers/yogi.py @manzilz
 /tensorflow_addons/optimizers/tests/yogi_test.py @manzilz
 
-/tensorflow_addons/rnn/cell.py @qlzh727 @pedrolarben
-/tensorflow_addons/rnn/tests/cell_test.py @qlzh727 @pedrolarben
+/tensorflow_addons/rnn/esn_cell.py @qlzh727 @pedrolarben
```

Comment:
For esn_cell and its test, you can put the owner to @pedrolarben, and the rest of them, you can leave them to me.

Reply:
applied 😄